tests/test_cifar_format.py

```diff
@@ -146,6 +146,11 @@ def test_can_import(self):
             DatasetItem(id='image_4', subset='test',
                 image=np.ones((32, 32, 3)),
                 annotations=[Label(2)]
+            ),
+            DatasetItem(id='image_5', subset='test',
+                image=np.array([[[1., 2., 3.], [4., 5., 6.]],
+                                [[1., 2., 3.], [4., 5., 6.]]]),
+                annotations=[Label(3)]
```

Comment:
Do we need a check about image orientation?

Reply:
What do you mean, how to check image orientation?